_concepts/program-summary.md

```diff
@@ -0,0 +1,55 @@
+---
+title: CFDA Program Summary
+
+archived_page: no
+archived_page_text:
+link_to_archived_designs: 
+link_to_active_designs:
+
+category: "Summary Pages:"
+
+headline: CFDA Program Summary
+
+copy:
+  - text: Below is an idea on how to display the details of a CFDA program. Please take a look and give us your feedback in the discussion section at the bottom of each tab.
+
+discussion_title: "Discussion suggestions:"
+
+discussion_bullet_points:
+- Does the layout and information help you complete your tasks?
+- Are the labels and terminology appropriate?
+- Is there other information you want to see?
+- What do you like and dislike?
+
+tabs_required: yes
+
+tabs:
+  - title: Version 0.1
+    url_hash: "#!version-0.1"
+    archived_text:
+    description: "Key Features:"
+    features:
+      - description: Description of the CFDA program
+      - description: Bar graph display total amount awarded for fiscal year
+      - description: Timeline of spending over time
+      - description: Breakdown of awards in program
+    disclaimer_text: <a class="usa-cta external-link" href='http://my36m8.axshare.com/#g=1&p=program_summary.html' target="_blank">View an interactive version of the below image</a>
+    image: "/assets/img/concept-program-summary.png"
```

Comment:
The image file had an underscore between `concept` and `summary` so I fixed it here.